pkg/operator/profilecalculator.go

```diff
@@ -464,6 +464,16 @@ func tunedRecommend(tunedSlice []*tunedv1.Tuned) []tunedv1.TunedRecommend {
 		return recommendAll[i].Priority != nil // undefined priority has the lowest priority
 	})
 
+	for i := 0; i < len(recommendAll)-1; i++ {
```

Comment:
Does `recommendAll` sorted by priority?
Like I can imagine the situation when you will have
[profile1, profile2, profile3] and
profile1.priority=30
profile2.priority=50
profile3.priority=30

Reply:
Yep, sorted by priority by the code above.